riscv_config/schemas/schema_isa.yaml

```diff
@@ -247,6 +241,50 @@ hart_schema:
       nullable: True
       default:
 
+    ###
+    #pte_ad_hw_update
+    #-----------------
+    #
+    # - **Description**: This is a boolean field indicating that when a virtual page is accessed 
+    #   and the A bit is clear, or is written and the D bit is clear, the implementation sets 
+    #   the corresponding bit(s) in the PTE in the hardware itself and does not raise an exception
+    #   for this particular scenario. When set to False, a corresponding page-fault exception needs
+    #   to be raised.
+    # - **Default**: False
+    # - **Constraints**: Can be set to True only if 'S' mode is implemented in the ISA
+    # - **Examples**:
+    #   
+    #   .. code-block:: yaml
+    #
+    #     pte_ad_hw_update: False
+    #     pte_ad_hw_update: True
+
+    pte_ad_hw_update: {type: boolean,  default: False, check_with: s_exists}
```

Comment:
Checking whether `satp.mode` field can take values other than `bare` should also be performed. Although this might be too complicated to achieve at this moment due to the warl nature of the field.